.github/stale.yml

```diff
@@ -0,0 +1,48 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - internal
+  - kind/bug
+  - kind/bug-qa
+  - kind/task
+  - kind/feature
+  - kind/design
+  - kind/ci-improvements
+  - kind/performance
+  - kind/flaky-test
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
```

Comment:
will this prevent all the issues in the unscheduled milestone from getting cleaned up?

Reply:
Yeah we probably need to get rid of that as it makes no sense.

Reply:
as long as we delete that milestone I think this makes sense

Reply:
@deniseschannon Can we drop the `Unscheduled` milestone?

Reply:
What do you mean by drop the Unscheduled milestone? In GH or where?

Reply:
Yes in GitHub

Reply:
I've deleted the milestone now. Thanks @paynejacob for considering how that would affect it!